src/Bridge/Symfony/Configuration.php

```diff
@@ -21,8 +21,14 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('cocur_slugify');
+        $treeBuilder = new TreeBuilder('cocur_slugify');
+
+        // Keep compatibility with symfony/config < 4.2
+        if (!\method_exists($treeBuilder, 'getRootNode')) {
```

Comment:
For a more clean way I'd reverse the condition to avoid negation.

Reply:
👍